src/ralph/accounts/views.py

```diff
@@ -28,6 +31,51 @@ def user_licence(self, item):
             return []
 
 
+class InventoryTagConfirmationView(RalphBaseTemplateView):
+    template_name = 'ralphuser/inventory_tag_confirmation.html'
+
+    def get_context_data(self, **kwargs):
+        print(self.request.method)
+        context = super().get_context_data(**kwargs)
+        asset_fields = [
+            ('barcode', _('Barcode / Inventory Number')),
+            'model__category__name', 'model__manufacturer__name',
+            'model__name', ('sn', _('Serial Number'))
+        ]
+
+        context['asset_details'] = MyEquipmentAssetList(
+            BackOfficeAsset.objects.filter(id=self.kwargs['asset_id']),
+            asset_fields,
+            request=self.request
+        )
+        return context
+
+
+class InventoryTagView(RalphBaseTemplateView):
+    template_name = 'ralphuser/inventory_tag.html'
+    
+    def post(self, request, *args, **kwargs):
```

Comment:
change it to ``def post(self, request, asset_id, *args, **kwargs)`` then you can access directly instead of ``request.POST['asset_id']``